yarn-project/cli-wallet/src/cmds/register_contract.ts

```diff
@@ -1,36 +1,18 @@
-import {
```

Comment:
This command was unnecessarily complicated, we can retrieve the instance from the node if already deployed (which is the use case)